locale/pl/LC_MESSAGES/amo.po

```diff
@@ -78,20 +76,19 @@ msgstr "Używane przez"
 
 #: src/amo/components/AddonMoreInfo.js:24
 msgid "Homepage"
-msgstr ""
+msgstr "Strona domowa"
 
 #: src/amo/components/AddonMoreInfo.js:32
 msgid "Support Site"
-msgstr ""
+msgstr "Strona pomocy"
 
 #: src/amo/components/AddonMoreInfo.js:38
 msgid "More information"
 msgstr "Więcej informacji"
 
 #: src/amo/components/AddonMoreInfo.js:42
-#, fuzzy
 msgid "Add-on Links"
-msgstr "Ikona dodatku"
+msgstr "Odnośniki dodatku"
```

Comment:
The heading covers the add-on's support url and homepage link so it's an umbrella title in this context if that makes sense?